tests/specs/mcd/end-subuu-pass-spec.k

```diff
@@ -5,7 +5,7 @@ module END-SUBUU-PASS-SPEC
 
     // End_subuu
     claim [End.subuu.pass]:
-      <k> #execute ~> CONTINUATION => #execute ~> CONTINUATION </k>
+      <k> #execute ~> CONTINUATION => JUMP JMPTO ~> #pc [ JUMP ] ~> #execute ~> CONTINUATION </k>
```

Comment:
The fixes to this proof and to `vat-subui` can be pulled out into its own PR that you can open up really quick (use a `git worktree` to do it in a separate directory). You can also then remove them from the list `tests/failing-symbolic.pyk` I believe?